File: projects/stock/src/main/java/com/vjcspy/spring/packages/stocksync/dto/vietstock/VietStockCredential.java
/* (C) 2024 */
package com.vjcspy.spring.packages.stocksync.dto.vietstock;

import lombok.Builder;
import lombok.Data;

@Builder
@Data
public class VietStockCredential {
    private String sid, rvt, vtsUsrLg, usrTk, csrf;
}

File: projects/stock/src/main/kotlin/com/vjcspy/spring/packages/stocksync/dto/vietstock/VietStockCredential.kt
/* (C) 2024 */
package com.vjcspy.spring.packages.stocksync.dto.vietstock

data class VietStockCredential(
    val sid: String,
    val rvt: String,
    val vtsUsrLg: String,
    val usrTk: String,
    val csrf: String,
)
